### ISSUE TITLE
interactive event: association should either be required or filter should be hidden

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/adhocracy-plus#1022]
**URL:** 
**user:** 
**expected behaviour:** 
**behaviour:** If no association is added in the dashboard, the list to filter for is empty.
**important screensize:**
**device & browser:** 
**Comment/Question:** We could either make the association required in the dashboard or hide the filter when it's not given. @lthueer @fredewe Which should it be? (P.S.: next release, though. :smile: )

![Screenshot from 2020-12-10 12-40-40](https://user-images.githubusercontent.com/8178179/101767872-2190ed80-3ae5-11eb-99a6-f52f6bf0230a.png)

**Comment by lthueer** [comment]
@fuzzylogic2000 I would go for the second option, hiding it when nothing is entered. Sometimes it may be useful to not add any affiliation. What do you think @fredewe ?

**Comment by fredewe** [comment]
I would say that it is a political decision and i would like to hear @hmkay s opinion on it.
In my opinion we can't force people to use the affiliation so the question would be if we lose people to other tools, when it is obligatory in a+ or will they maybe come up with not useful affiliations when we make them required?
So i would be a little in favor for the second option even though i think they are a key feature of  the interactive event and we should strongly recommend organizations to use them.

**Comment by hmkay** [comment]
uh, yes, I also think it is a political discussion or decision.

One unique feature of SpeakUp was that we made it required to have affiliations. This was because there are always affiliations and categories at hand on which base  moderators decide which question to pick or not to pick. We deliberately wanted to make that more visible and to make people think about it, whom they would like to invite into a discussion. In the end, we invite moderators to reflect on what they think, whom they´d like to hear and whom they usually hear in discussions. Only recommending it wouldn´t be enough in my opinion, people willl simply not use it and stick to their unconscious decisions on whom to include into the discussion.

It therefore is also education for more inclusive discussions.

Because of this, I am strongly in favour of making it obligatory. I do not think that we loose people to other tools. We have THE argument that people can visualise their whole process, their whole participation. This is something no other tool can do. Nevertheless, in this option, we do have to educate people on how to name affiliations I guess.

What do the others think?

**Comment by fuzzylogic2000** [comment]
Yes, I think @hmkay is right! Also our statistics do not make sense, when there is no affiliation. Or do they now after the statistics-story?
So, what's the decision @fredewe @lthueer ? Should we hide it or make it required?
One more thing we could find out if we want more info before we decide, would be how it was actually used on prod, meaning if there were any projects that didn't fill in the affiliation. If there weren't any, I think, just making it required would be fine! I could also find out if you want me to @fredewe @lthueer 
And one more question: is this also an issue in mB, @CarolingerSeilchenspringer ? And if it isn't, how do we handle it?

**Comment by fredewe** [comment]
There are organisations using it without affiliations but i would say: Let us make it required for now. We can change it later when we receive bad feedback. :)

**Comment by fuzzylogic2000** [comment]
Nice! Then, we also need a value for the events that don't currently have an affiliation...
What could that be and makes sense? "participant" maybe?
@fredewe @lthueer

**Comment by fredewe** [comment]
I would be fine with "participant". In German it would be "Teilnehmer*in"?

**Comment by hmkay** [comment]
"Teilnehmy" :D

**Comment by hmkay** [comment]
https://media.giphy.com/media/y8Mz1yj13s3kI/giphy.mp4

**Comment by fredewe** [comment]
https://www.youtube.com/watch?v=E7Vn_pS2G-Y

**Comment by fuzzylogic2000** [comment]
@fredewe @hmkay Unluckily we didn't need a default after all. :( 
We can only make it required in the dashboard, not on model level, because it's the same as the categories for other modules (meaning using the same model/table in the db).
So, new interactive events can only be added to the project and published, when there are affiliations. Old ones stay like they are, but will have the red mark next time the initiator goes to the dashboard. They can just ignore that.
